lhapdf.spec

```diff
@@ -1,27 +1,25 @@
-### RPM external lhapdf 6.2.1
-%define setsversion 6.2.1a_2
+### RPM external lhapdf 6.4.0
+%define setsversion 6.4.0e
 
-Source: http://www.hepforge.org/archive/lhapdf/LHAPDF-6.2.1.tar.gz
+Source: http://www.hepforge.org/archive/lhapdf/LHAPDF-%{realversion}.tar.gz
 
 Source1: lhapdf_makeLinks
 
-Source2: http://www.hepforge.org/archive/lhapdf/pdfsets/6.1/cteq6l1.tar.gz
-Source3: http://www.hepforge.org/archive/lhapdf/pdfsets/6.1/CT10.tar.gz
-Source4: http://www.hepforge.org/archive/lhapdf/pdfsets/6.1/MSTW2008nlo68cl.tar.gz
-Source5: https://www.hepforge.org/archive/lhapdf/pdfsets/6.1/MMHT2014lo68cl.tar.gz
-Source6: https://www.hepforge.org/archive/lhapdf/pdfsets/6.1/MMHT2014nlo68cl.tar.gz
+Source2: http://www.hepforge.org/archive/lhapdf/pdfsets/6.1/MSTW2008nlo68cl.tar.gz
 
-Source7: lhapdf_pdfsetsindex
+Source3: lhapdf_pdfsetsindex
 
-Requires: python
-BuildRequires: py2-cython
+Requires: python3
+BuildRequires: py3-cython
 
 %define keep_archives true
 
 %prep
 %setup -q -n LHAPDF-%{realversion}
 
-./configure --prefix=%{i} 
+PYTHON=$(which python3) \
```

Comment:
@aandvalenzuela , also remove this line or use `PYTHON=$(which python)` . For 10.2.X , we still have python ( i.e. python2) as default